api/app/src/main/kotlin/packit/model/dto/RolesAndUsers.kt

```diff
@@ -0,0 +1,14 @@
+package packit.model.dto
+
+import packit.model.Role
+import packit.model.User
+
+data class RolesAndUsers(
```

Comment:
Given that Users are Roles, can't we just use a `List<Role>` here that encompasses both of them?

The whole benefit of having all Users be Roles is to avoid code duplication, we should take advantage of that.

Reply:
That is what was returned originally... but the FE was required to mangle the data to get a list of roles and users (this refactor is trying to address this issue)... thus it made more sense to create a class that encompasses both of them

Reply:
I think we should make two requests, one to `/users` and one to `/roles`. Not mash the two together.